app/assets/javascripts/app/form-validation.js

```diff
@@ -2,7 +2,6 @@ import h5f from 'h5f';
 
 import validateField from './validate-field';
 
-
```

Comment:
oh interesting -- do we have a linter for this? I have not seen that style before